config.py

```diff
@@ -1470,6 +1470,30 @@ def get_t1_from_meeg(bids_path):
     ```
 """
 
+source_info_update: Optional[Dict[str, str]] = dict(suffix='ave')
+"""
+Match the file name from where to read the info in forward and inverse solution
+computation by updating 'bids_path'.
+
+???+ example "Example"
+    ```python
+    source_info_update = {'processing': 'clean',
+                          'suffix': 'epo'}
+    ```
+"""
+
+inverse_targets: Optional[List[str]] = []
+"""
+List of the different types of data on which to apply the inverse operator.
+
+If [] only the inverse solution coputation is done.
```

Comment:
I would make it `None` in this case … `Optional[...]` in fact implies that `None` is an accepted value